modules/apps/knowledge-base/knowledge-base-api/src/main/java/com/liferay/knowledge/base/service/KBArticleServiceWrapper.java

```diff
@@ -217,14 +218,14 @@ public com.liferay.knowledge.base.model.KBArticle getKBArticle(
 
 	@Override
 	public String getKBArticleRSS(
-			long resourcePrimKey, int status, int rssDelta,
-			String rssDisplayStyle, String rssFormat,
-			com.liferay.portal.kernel.theme.ThemeDisplay themeDisplay)
+			String displayStyle, int max, long resourcePrimKey, int status,
+			com.liferay.portal.kernel.theme.ThemeDisplay themeDisplay,
+			String type, double version)
```

Comment:
Same as before. The sorting criteria is not clear to me. IMO, `resourcePrimKey` should be first, and `themeDisplay` the last.

Reply:
Done as above